<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stkPivotCalibrationWidget</class>
 <widget class="QWidget" name="stkPivotCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stkPivotCalibrationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>工具名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="toolDescriptionLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="computeCalibrationButton">
       <property name="text">
        <string>开始标定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="completeCalibrationButton">
       <property name="text">
        <string>完成标定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="outputEdit">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
